js/modules.js

```diff
@@ -541,7 +541,7 @@ new Module("Functions")
 .addAction("endfunction","Ends a function block")
 .addAction("[value =] call(<functionname>,[...parameters])","Calls the function")
 .addAction("[value =] <functionname>([...parameters])","Alias to [value =] call(<functionname>,[...parameters])")
-.addAction("return(<value>)","Sets the return value of the function and stops it. Return value can be a constant, a variable, a comma separated list or an array")
+.addAction("return(<value>,[value 2,value 3...])","Sets the return value of the function and stops it. Return value can be an integer, a string, an expanded variable, an array variable or a comma separated list.")
 .addIterator("foreach(\"functions\")","Possible variables: %FUNCTIONNAME%")
```

Comment:
Constant includes booleans

Reply:
yeah, my bad, I couldn't see the "expanded" in your edited message (no idea why, I'm pretty sure I reloaded the page) so I wasn't sure what you meant

\~ now it all makes sense \~